### PR TITLE
net-dns/mdns-repeater: new package

### PR DESCRIPTION
**Pull request by cernekee**
This program forwards multicast DNS (RFC6762) traffic between subnets.
Tested on Chrome OS, amd64 and arm boards.

Package-Manager: portage-2.2.12-r9

### CODE REVIEW

File: net-dns/mdns-repeater/mdns-repeater-20110921.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=4
```
Comment (SoapGentoo) [inline net-dns/mdns-repeater/mdns-repeater-20110921.ebuild]: `EAPI=4`, we won't accept this. Move to `EAPI=6` please

File: net-dns/mdns-repeater/mdns-repeater-20110921.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+inherit eutils toolchain-funcs
```
Comment (SoapGentoo) [inline net-dns/mdns-repeater/mdns-repeater-20110921.ebuild]: why do you need eutils?

File: net-dns/mdns-repeater/mdns-repeater-20110921.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+DESCRIPTION="Multicast DNS repeater"
+HOMEPAGE="https://bitbucket.org/geekman/mdns-repeater/"
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="amd64 arm"
```
Comment (doughdemon) [inline net-dns/mdns-repeater/mdns-repeater-20110921.ebuild]: We do not commit straight to stable.

File: net-dns/mdns-repeater/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>cernekee@chromium.org</email>
+		<name>Kevin Cernekee</name>
+	</maintainer>
+	<upstream>
```
Comment (SoapGentoo) [inline net-dns/mdns-repeater/metadata.xml]: Please also add the proxy maintainers project below your person

**Comment by monsieurp** [comment]
LGTM.

**Pull request closed without merging**